screen/import-data/src/main/java/com/ivy/importdata/csv/domain/CSVImporterV2.kt

```diff
@@ -108,7 +110,11 @@ class CSVImporterV2 @Inject constructor(
                 0.0
             }
             onProgress(0.5 + progressPercent / 2)
-            transactionWriter.save(transaction.toEntity())
```

Comment:
For example here, instead of using this conversion directly, i can use the suspend  extension function that i created.